### DISCUSSION TITLE
중고거래 페이지에서 판매중/판매완료 처리

### DISCUSSION THREAD

**Discussion by JangGusWjd** [MajorIn-ABO/MajorIn_FE#19] [Q&A]
중고거래 페이지에서 판매 중인 교재만 보이게 하는게 좋을까 아니면 판매 완료된 교재도 보이게 하는게 좋을까?
판매 완료된 교재는 중고 거래 페이지에서는 안보이게 하고 마이 페이지에서 자신이 판매한(판매했던) 책 리스트로는 보이게 할까?
아니면 판매 완료된 교재도 가격 책정 등의 이유로 중고 거래 페이지에서 리스트에 계속 보이게할까?

**Comment by cho-ja-young** [comment]
판매완료된건 그래도 정보는 남아있는게 좋을 것 같아서 흑백처리(? 하고 일단 리스트에 남겨두거나 
아니면 필터링으로 판매완료된 책. 약간 이렇게 빼는것도 좋을듯!

**Comment by wlsl10** [comment]
웅 나도 책이 현재 판매되는 게 없으면 시세가 어땠는지를 알기 어려우니까 정보가 남아있는 게 나을 것 같아! 자영이 의견처럼 필터링하면 제일 좋을 것 같긴 해!
